benchmark/variable_benchmark.cpp

```diff
@@ -143,4 +143,28 @@ static void BM_Variable_binary_with_VariableProxy(benchmark::State &state) {
 BENCHMARK(BM_Variable_binary_with_Variable);
 BENCHMARK(BM_Variable_binary_with_VariableProxy);
 
+static void BM_Variable_assign_1d(benchmark::State &state) {
+  const auto size = state.range(0);
+
+  const auto a = createVariable<double>(Dims{Dim::X}, Shape{size});
+  auto b = createVariable<double>(Dims{Dim::X}, Shape{size});
```

Comment:
createVariable -> makeVariable

Reply:
I thought `createVariable` was the correct function to use now? Has this changed again?

Reply:
Oh. Apparently it had.